projects/storefrontapp-e2e-cypress/cypress/integration/regression/multisite-isolation/multisite-isolation.core-e2e-spec.ts

```diff
@@ -0,0 +1,109 @@
+import { clearAllStorage } from '../../../support/utils/clear-all-storage';
+import { user } from '../../../sample-data/checkout-flow';
+import { register } from '../../../helpers/auth-forms';
+import * as login from '../../../helpers/login';
+import * as alerts from '../../../helpers/global-message';
+
+import {
+  ELECTRONICS_BASESITE,
+  ELECTRONICS_CURRENCY,
+} from '../../../helpers/checkout-flow';
+import { POWERTOOLS_BASESITE } from '../../../sample-data/b2b-checkout';
+
+export const ELECTRONICS_STANDALONE_BASESITE = 'electronics-standalone';
+export const POWERTOOLS_STANDALONE_BASESITE = 'powertools-standalone';
+
+describe('Multisite Isolation', () => {
+  before(() => {
+    cy.window().then((win) => {
+      win.localStorage.clear();
+    });
+  });
+
+  beforeEach(() => {
+    clearAllStorage();
+  });
+
+  describe('Standalone authorization', () => {
+    beforeEach(() => {
+      Cypress.env('BASE_SITE', ELECTRONICS_STANDALONE_BASESITE);
+
+      cy.cxConfig({
+        context: {
+          baseSite: [ELECTRONICS_STANDALONE_BASESITE],
+          currency: [ELECTRONICS_CURRENCY],
+        },
+      });
+    });
+
+    it.skip('should register customer on standalone environment', () => {
```

Comment:
Are tests in this suite `skip`ped deliberately?

Reply:
It requires change in CI sample data to working properly, so that's why I decided to skip them for now.
I have a ticket in progress: CXSPA-2035 related to this issue.